src/datasetManifest.js

```diff
@@ -286,44 +232,23 @@ export const allDatasets = [
     metadataPath: "datasets/movies.json",
     imagePath: "datasets/temp.jpg"
   },
-  {
-    id: "mushrooms",
-    name: "Poisonous Mushrooms",
-    path: "datasets/mushrooms.csv",
-    metadataPath: "datasets/mushrooms.json",
-    imagePath: "datasets/mushrooms.jpg"
-  },
   {
     id: "car_evaluation",
     name: "Car Evaluation",
     path: "datasets/car_evaluation.csv",
     metadataPath: "datasets/car_evaluation.json",
     imagePath: "datasets/car_evaluation.jpg"
   },
-  {
-    id: "school_scores",
-    name: "SAT Scores",
-    path: "datasets/school_scores.csv",
-    metadataPath: "datasets/school_scores.json",
-    imagePath: "datasets/school_scores.jpg"
-  },
-  {
-    id: "house-votes-84",
-    name: "1984 US House Votes",
-    path: "datasets/house-votes-84.csv",
-    metadataPath: "datasets/house-votes-84.json",
-    imagePath: "datasets/house-votes-84.jpg"
-  }
 ];
 
 export function getAvailableDatasets(specificDatasets) {
-  if (specificDatasets && specificDatasets.length > 1) {
-    return allDatasets.filter(dataset => {
-      return specificDatasets.includes(dataset.id);
-    });
-  } else {
-    return allDatasets;
-  }
+  // Filter datasets by a specified array (in Levelbuilder) or filter out toy datasets.
+  let availableDatasets = (specificDatasets && specificDatasets.length > 1) ?
+    allDatasets.filter(dataset => specificDatasets.includes(dataset.id)) :
+    allDatasets.filter(dataset => dataset.toy !== true);
```

Comment:
Maybe just `! dataset.toy`?

Reply:
Ahhh :sweat_smile: Thank you for catching this!